base/applications/utilman/umandlg/process.c

```diff
@@ -211,7 +211,7 @@ BOOL CloseProcess(IN LPCWSTR lpszProcessName)
     hProcess = OpenProcess(PROCESS_TERMINATE, FALSE, dwProcessID);
     if (!hProcess)
     {
-        DPRINT("CloseProcess(): Failed to open the process for termination! (Error: %lu)", GetLastError());
+        DPRINT("CloseProcess(): Failed to open the process for termination! (Error: %lu)\n", GetLastError());
```

Comment:
```suggestion
        DPRINT("Failed to open the process for termination! (Error: %lu)\n", GetLastError());
```
DPRINT logs the function name itself already.

Reply:
no it doesn't.

Reply:
Indeed, it doesn't. My fault. Resolved.